Dockerfile

```diff
@@ -258,7 +274,7 @@ RUN set -ex; \
 
 RUN for file in $(find /usr/share/postgresql -name 'postgresql.conf.sample'); do \
         # We want timescaledb to be loaded in this image by every created cluster
-        sed -r -i "s/[#]*\s*(shared_preload_libraries)\s*=\s*'(.*)'/\1 = 'timescaledb,\2'/;s/,'/'/" $file \
+        sed -r -i "s/[#]*\s*(shared_preload_libraries)\s*=\s*'(.*)'/\1 = 'vectors,timescaledb,\2'/;s/,'/'/" $file \
```

Comment:
Won't this result in postgres not starting when vectors is missing?

Reply:
Yes, you are right, totally missed that. Is there a recommended way to configure `shared_preload_libraries` for the additional extensions?

Reply:
I think that should generally be left to the users of the image to supply a postgresql.conf that includes whatever they want.

Reply:
Good point. Is there documentation somewhere, that I might have missed, which would need updating in this regard?

Reply:
Not really? This image follows the same basic methods as the [postgresql docker image database configuration](https://www.docker.com/blog/how-to-use-the-postgres-docker-official-image/#4-Database-configuration) where we pass the command-line arguments to `postgres`, so you can add `-c` to set run-time parameters, or you could use docker's `-v` to map a configuration file to the container. I didn't really find where overriding `shared_preload_libraries` is documented.

I did test that this works though:
`docker run --rm -it -e POSTGRES_HOST_AUTH_METHOD=trust timescale/timescaledb-ha:pg16-ts2.13 -c shared_preload_libraries=pg_stat_statements,timescaledb`

Reply:
Just wanted to check if I missed updating relevant documentation. Thanks :)